### DISCUSSION TITLE
A question ablout launch_menu args

### DISCUSSION THREAD

**Discussion by r-xiumu** [wezterm/wezterm#2777] [Q&A]
**(Wezterm is the most awesome terminal I uesed}**  

I want to set "ssh login" by launch_menu.
My command:
`ssh user@ip -o KexAlgorithms=diffie-hellman-group1-sha1 -o PubkeyAcceptedAlgorithms=ssh-rsa -o HostkeyAlgorithms=ssh-rsa  ` 

launch_menu args:
```
launch_menu = {
	{
          label = 'test',
          args = {
	        'ssh',
		'user@ip',
		'-o',
		'KexAlgorithms=diffie-hellman-group1-sha1',
		'-o',
		'PubkeyAcceptedAlgorithms=ssh-rsa ',
		'-o',
		'HostkeyAlgorithms=ssh-rsa '
	  }	  
}
```
Input command will hint for  passwd, but by launch_menu(select test) nothing happened.
I don't understand.
Please help me. ^_^

**Comment by wez** [comment]
Try setting `exit_behavior = "CloseOnCleanExit"` so that you can see any error messages that your ssh invocation may emit.
Hint: you have trailing spaces on some of the arguments that I think will prevent it from working how you want.

https://wezfurlong.org/wezterm/config/lua/config/exit_behavior.html

**Reply by r-xiumu** [reply]
Thank you.
I saw error message (`Bad configuration option:PubkeyAcceptedAlgorithms`) by setting exit_behavior.

For my openssh version, it should be PubkeyAcceptedKeyTypes.
